apis/network/v1alpha3/securitygroup_types.go

```diff
@@ -45,68 +45,174 @@ func (v *VPCIDReferencerForSecurityGroup) Assign(res resource.CanReference, valu
 		return errors.New(errResourceIsNotSecurityGroup)
 	}
 
-	sg.Spec.VPCID = value
+	sg.Spec.VPCID = aws.String(value)
 	return nil
 }
 
+// SecurityGroupParameters define the desired state of an AWS VPC Security
+// Group.
+type SecurityGroupParameters struct {
```

Comment:
I had a bit of trouble following the changes in this file due to how github is showing the diff, but it this still additive or should these changes require a bump to `v1alpha4`?

Reply:
It should be additive. I'm testing it with Wordpress stack that has v1alpha3 `SecurityGroup` resources.